src/utilities/modules/vaccines.js

```diff
@@ -0,0 +1,41 @@
+import { formattedDifferenceBetweenDates, formatExposureRange } from '../formatters';
+
+/**
+ * Method for creating data for BreachTable rows.
+ * Accepts an array (Example below) of objects,
+ * each having a breach property which contains
+ * a realm results of sensorLog objects.
+ * @param {Array} data Example below
+ * [ {
+ *     breach: [ sensorLogs ],
+ *     ...
+ *   },
+ *    ...
+ * ]
+ */
+export const setBreachData = (data = []) => {
+  const dataCopy = [];
+  data.forEach(dataSet => {
+    const { breach } = dataSet;
+    // A breach must have a minimum of 3 sensorlogs to be considered
+    // a breach.
+    if (!breach || breach.length < 4) return;
+    const [firstSensorLog = null, , { timestamp: lastTimestamp } = {}] = breach;
```

Comment:
that's cool, didn't know `, ,` lets you deconstruct first and last, why null for fistSensorLog and { timestamp: lastTimestamp } = {} for last one, why not both ?

Reply:
Needed the timestamp from the last one but other things from the first one

Reply:
as you've suggested, extraAggregation in sensorLogsExtractBreaches should resolve the need for this